libs/components/src/lib/note/note.scss

```
@@ -0,0 +1,55 @@
+@use '../../../../shared/src/lib/sass/mixins/typography' as typography;
+@use '../../../../../node_modules/@vonage/vivid-tokens/dist/sass/themes/consts' as theme-consts;
+@use '../../../../shared/src/lib/sass/mixins/connotation/config' with (
+	$connotations: success info alert announcement warning,
+	$default: announcement,
+);
+@use '../../../../shared/src/lib/sass/mixins/connotation' as connotation;
+
+.base {
+	@include connotation.connotation;
+
+	display: flex;
+	align-items: stretch;
+	padding: 20px;
+	border-left: 8px solid var(#{connotation.$vvd-color-connotation});
+	background-color: var(#{theme-consts.$vvd-color-canvas});
+	border-radius: 6px;
+	box-shadow: inset 0 1px 0 0 var(#{theme-consts.$vvd-color-neutral-40}),
+		inset -1px 0 0 0 var(#{theme-consts.$vvd-color-neutral-40}),
+		inset 0 -1px 0 0 var(#{theme-consts.$vvd-color-neutral-40});
+	color: var(#{theme-consts.$vvd-color-on-canvas});
+	@supports(contain: content) {
+		contain: content;
+	}
+	@supports not (contain: content) {
+		overflow: hidden;
+	}
+}
+
+.note-icon {
+	#{connotation.$vvd-color-connotation}: currentColor;
+
+	margin-inline-end: 16px;
```

add this to match the design:
```suggestion
	margin-inline-end: 16px;
	align-self: flex-start;
  font-size: 24px;
```